docs/upgradeProcess.md

```diff
@@ -0,0 +1,204 @@
+
+CIS Features and Examples
+-------------
+
+| Feature                              | Example  | CIS Version |
+|-----------|-------------|---------------|
+| user-defined-as3                     | [Example](https://clouddocs.f5.com/containers/latest/userguide/config-map.html#modify-as3-configmap) | v2.0.0      |
+| override-as3                         | [Example](https://clouddocs.f5.com/containers/latest/userguide/config-map.html#override-as3-configmap) | v2.0.0      |
+| VirtualServer With TLS profile       | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServerWithTLSProfile) | v2.1.0      |
+| virtualServer insecure               | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServer/unsecure-virtual-server) | v2.1.1      |
+| Health Monitors in EDNS              | [Example](https://raw.githubusercontent.com/F5Networks/k8s-bigip-ctlr/master/docs/config_examples/customResource/ExternalDNS/externaldns-tcp-monitor.yaml) | v2.2.0      |
+| virtualServer-waf                    | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServer/waf) | v2.2.0      |
+| virtualServer Multiport Services     | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServer/MultiPortServices) | v2.2.0      |
+| TransportServer with IPAM Label      | [Example](https://raw.githubusercontent.com/F5Networks/k8s-bigip-ctlr/master/docs/config_examples/customResource/TransportServer/transport-server-with-ipamLabel/transport-with-ipamLabel.yaml) | v2.4.0      |
+| TransportServer with Service address | [Example](https://raw.githubusercontent.com/F5Networks/k8s-bigip-ctlr/master/docs/config_examples/customResource/TransportServer/ts-with-service-address.yaml) | v2.4.0      |
+| IngressLink with IPAM Label          | [Example](https://raw.githubusercontent.com/F5Networks/k8s-bigip-ctlr/master/docs/config_examples/customResource/IngressLink/ingressLink-with-ipamLabel/ingresslink-with-ipamLabel.yaml) | v2.6.0      |
+| virtualServer with HostGroup         | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServer/virtual-with-hostGroup) | v2.7.0      |
+| virtualServer with Wildcard domain   | [Example](https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/customResource/VirtualServer/virtual-with-wildcard-domain) | v2.7.0      |
+| Policy-CRD                           | [Example](https://raw.githubusercontent.com/F5Networks/k8s-bigip-ctlr/master/docs/config_examples/customResource/Policy/sample-policy.yaml) | v2.7.0      |
+| Filter-Tenants                       | [Example](https://clouddocs.f5.com/containers/latest/userguide/config-map.html#filter-tenant-support-for-as3-configmap) | v2.7.1      |
```

Comment:
we are supporting filter-tenants from 2.7.0

Reply:
Updated